xikolo-common/Data/Helper/CoreDataHelper.swift

```diff
@@ -67,4 +67,26 @@ class CoreDataHelper {
         }
     }
 
+    static func clearCoreDataStorage() {
+        clearCoreDataEntity("Course")
+        clearCoreDataEntity("CourseSection")
+        clearCoreDataEntity("CourseItem")
+        clearCoreDataEntity("Content")
+        clearCoreDataEntity("Video")
+        clearCoreDataEntity("RichText")
+        clearCoreDataEntity("Quiz")
+        clearCoreDataEntity("NewsArticle")
```

Comment:
I did. But after some research I couldn't find a better method. Only this improvement.